tools/releasing/packaging/Arch_User_Repository/PKGBUILD

```
@@ -1,29 +1,37 @@
 # Maintainer:  Florian Lindner <florian.lindner@xgm.de>
 
 pkgname=precice
-pkgver=1.6.1
+pkgver=2.1.0
 pkgrel=1
 pkgdesc="A Coupling Library for Partitioned Multi-Physics Simulations on Massively Parallel Systems"
 arch=('x86_64')
 url="https://www.precice.org"
 license=('LGPL3')
-depends=('openmpi' 'boost')
+depends=('boost' 'libxml2' 'openmpi' 'petsc' 'python-numpy')
 makedepends=('cmake' 'eigen')
-optdepends=('')
+optdepends=()
 provides=('precice')
-source=("https://github.com/precice/precice/archive/v${pkgver}.tar.gz")
-sha256sums=('7d0c54faa2c69e52304f36608d93c408629868f16f3201f663a0f9b2008f0763')
+source=("$pkgname-$pkgver.tar.gz::https://github.com/precice/precice/archive/v${pkgver}.tar.gz")
+sha256sums=('1e6432724f70d0c6c05fdd645e0026754edbc547719a35bf1d3c12a779b1d00e')
```

Let's remove the checksums from here. They are anyway supposed to be generated automatically when running `makepkg`, before publishing to AUR.
```suggestion
sha256sums=()
```